lib/charliecloud.py

```diff
@@ -190,8 +190,10 @@ class Download_Mode(enum.Enum):
 /// Common ///
 
 option: "--" OPTION_KEY "=" OPTION_VALUE
+option_keypair: "--" OPTION_KEY "=" OPTION_VAR "=" OPTION_VALUE
 OPTION_KEY: /[a-z]+/
-OPTION_VALUE: /[^ \t\n]+/
+OPTION_VALUE: /[^= \t\n]+/
```

Comment:
I think you can actually leave `OPTION_VALUE` unchanged.

Reply:
the `=` is so it is able to find multiple keypair options. When `OPTION_VALUE` is unchanged the 2nd/3rd/etc... options don't get parsed the correct way. Is there a different way to get that working to have `OPTION_VALUE` unchanged?